### PR TITLE
Basp-MR-96: Classes add remaining redux methods

### PR DESCRIPTION
**Pull request by SantiH97**
Added the following redux fetch methods
-POST
-PUT
-DELETE

Added CSS fixes

### CODE REVIEW

File: src/Components/Classes/Form/index.jsx [hunk 1]
```diff
@@ -4,16 +4,20 @@ import { Select } from '../../Shared/Inputs/index';
 import Button from '../../Shared/Button/index';
 import { useParams, useHistory } from 'react-router-dom';
 import Modal from '../../Shared/Modal';
+import { postClass, putClass } from '../../../redux/classes/thunks';
+import { useSelector } from 'react-redux';
+import { useDispatch } from 'react-redux';
+import Loader from '../../Shared/Loader/';
+import { getActivities } from '../../../redux/activities/thunks';
```
Comment (DelfiLT) [inline src/Components/Classes/Form/index.jsx:11]: You have 3 different imports from thunks, use one

File: src/Components/Classes/Form/index.jsx [hunk 2]
```diff
@@ -4,16 +4,20 @@ import { Select } from '../../Shared/Inputs/index';
 import Button from '../../Shared/Button/index';
 import { useParams, useHistory } from 'react-router-dom';
 import Modal from '../../Shared/Modal';
+import { postClass, putClass } from '../../../redux/classes/thunks';
+import { useSelector } from 'react-redux';
+import { useDispatch } from 'react-redux';
```
Comment (DelfiLT) [inline src/Components/Classes/Form/index.jsx:9]: You have 2 different imports from react redux, use one

File: src/Components/Classes/Form/index.jsx [hunk 3]
```diff
@@ -369,18 +323,17 @@ const Form = () => {
           <h2>{id ? 'Edit' : 'Add'}</h2>
           {showSuccessModal && (
```
Comment (DelfiLT) [inline src/Components/Classes/Form/index.jsx:324]: The success modal it's not working, you have to use it when it redirects you in the index

File: src/redux/classes/actions.js [hunk 4]
```diff
@@ -1,21 +1,99 @@
-import { GET_CLASSES_PENDING, GET_CLASSES_SUCCESS, GET_CLASSES_ERROR } from './constants';
+import {
```
Comment (DelfiLT) [inline src/redux/classes/actions.js:1]: For a suggestion you could use import * to not call one by one

**Review by DelfiLT — CHANGES_REQUESTED** [review]
Functionallity:
The success modals aren't working

UX/UI:
For the delete modal you should use the warning modal, and the delete button should be red, you should use the delete button

**Review by DelfiLT — APPROVED** [review]

**Review by Bigzufi — APPROVED** [review]
Good Job!!

**Comment by SantiH97** [comment]
[Classes - Fix- Delete modal and error handling](https://trello.com/c/8Nema4C0/103-classes-fix-delete-modal-and-error-handling)

**Pull request merged**